go.mod

```diff
@@ -120,7 +120,7 @@ require (
 	github.com/prometheus/client_golang v1.16.0
 	github.com/prometheus/client_model v0.4.0
 	github.com/prometheus/common v0.44.0
-	github.com/rancher/aks-operator v1.3.0-rc11
```

Comment:
this shouldn't happen. I think its because shepherd go.mod isn't updated to 11 yet.